extensions/micrometer/runtime/src/main/java/io/quarkus/micrometer/runtime/MicrometerCountedInterceptor.java

```diff
@@ -108,4 +110,16 @@ private void record(Counted counted, Tags commonTags, Throwable throwable) {
     private Tags getCommonTags(String className, String methodName) {
         return Tags.of("class", className, "method", methodName);
     }
+
+    private MicrometerCounted getCounted(InvocationContext context) {
```

Comment:
I had a common method for this lookup elsewhere...

Reply:
I added a new method to the `ArcInvocationContext`...